pkg/filter/proxy/masterslavereader.go

```diff
@@ -23,40 +23,45 @@ import (
 )
 
 type (
-	// masterSlaveReader reads bytes to master,
+	// primarySecondaryReader reads bytes to master,
 	// and synchronize them to slave.
 	// Currently only support one slave.
-	masterSlaveReader struct {
-		masterReader io.Reader
-		slaveReader  io.Reader
+	primarySecondaryReader struct {
```

Comment:
It seems no code is using this struct, if yes, please remove it, if no, please update the comment according to the rename.

Reply:
It is not used indeed, let's delete it